spec/requests/api/moves_controller_index_spec.rb

```diff
@@ -208,38 +208,6 @@
           end
         end
       end
-
-      context 'when the move includes both a profile id and a person id' do
```

Comment:
We validate the inclusion of a person, here: 

https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/dbf4d75b6ccc0ca24ef14640f92394532c4e1653/spec/serializers/move_serializer_spec.rb#L70-L106